src/ui/css/fontello/css/fontello.css

```diff
@@ -74,5 +74,6 @@
 .icon-bell:before { content: '\e811'; } /* '' */
 .icon-bell-off-empty:before { content: '\e812'; } /* '' */
 .icon-attach:before { content: '\e813'; } /* '' */
+.icon-emoji:before { content: '\1f603'; font-family: "Helvetica Neue"} /* '' */
```

Comment:
I didn't think it would fit to use Noto here, since the "attach" icon is monochromatic also.